### DISCUSSION TITLE
模组下载界面希望记住之前的进度

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#1011] [提问] [answered]
# 具体情况
安装模组的时候经常遇到要下载前置模组,和主要模组,
切换到前置模组页面选择版本的时候,
之前的进度又要重新滑动才能回到之前的状态
比如下载一个旧版本比如游戏版本1.12.2的模组,需要滑动到下面,但是看到需要前置模组,于是点击前置模组界面,下载完前置模组后,返回下载主要模组,又要重新滑动到下面,希望可以记住之前进度,减少重复滑动.

**Comment by WForst-Breeze** [comment, marked as answer]
或许可以交一个 **[优化建议](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E4%BC%98%E5%8C%96%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=ch.yml)**，看看龙猫怎么看
当您在 **mod 搜索**界面指定了目标版本后，PCL2 会使得被指定的版本在选定 mod 版本列表中置顶，或许应该够用，如下图
![image](https://user-images.githubusercontent.com/110760354/214778257-8f5fdf74-3443-4bf4-bea9-64e526ec5cb2.png)
【在本图中，我在搜索界面选定了目标版本：1.8，此时 JEI 的 1.8 版本被置顶】

**Reply by ghost** [reply]
不错不错 这样就解决了 或许可以根据当前游戏版本自动进行置顶? 更方便

**Reply by ghost** [reply]
你那边可以换个标签吗 
 不想再打字了

**Reply by WForst-Breeze** [reply]
> 不错不错 这样就解决了 或许可以根据当前游戏版本自动进行置顶? 更方便

PCL2 目前倒是可以根据被选定的 Mod 的游戏版本和模组加载器来自动将您想要的 Mod 下载到相对应的游戏版本中去

> 你那边可以换个标签吗
不想再打字了

如果您想要您提及的功能的话可以直接复制粘贴交到 Issues 里，在 Discussion 里的建议我们没有办法加 Label 或者是进行相关处理，抱歉啦……qwq

**Comment by miner233** [comment]
加个搜索其实也可以
但是我之前发 issues 建议龙猫加个搜索，龙猫拒绝了
